.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+- repo: https://github.com/PyCQA/flake8
+  rev: 6.0.0  # 6.0.0 drops Python 3.7 support
```

Comment:
If it drops 3.7 support shouldn't we use an older version? The same applies to `isort`.